DotNetHowTo/DotNetHowTo/Address.Methods.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace AzureSearch.SDKHowTo
+{
+    using System;
+    using System.Text;
+
+    public partial class Address
+    {
+        // This implementation of ToString() is only for the purposes of the sample console application.
+        // You can override ToString() in your own model class if you want, but you don't need to in order
+        // to use the Azure Search .NET SDK.
+        public override string ToString()
+        {
+            var builder = new StringBuilder();
+
+            if (!IsEmpty)
+            {
+                builder.AppendFormat("{0}\n{1}, {2} {3}\n{4}", StreetAddress, City, StateProvince, PostalCode, Country);
+            }
+
+            return builder.ToString();
+        }
+
+        public bool IsEmpty {
```

Comment:
Since C# 7, you can declare properties using expression-bodied syntax:

```csharp
public bool IsEmpty =>
    String.IsNullOrEmpty(StreetAddress) && String.IsNullOrEmpty(City) &&
    String.IsNullOrEmpty(StateProvince) && String.IsNullOrEmpty(PostalCode) &&
    String.IsNullOrEmpty(Country);
```

Reply:
Also, you'll probably need to mark this with `[JsonIgnore]` if you're using `FieldBuilder`, otherwise it will get included in your index schema and weird things will happen.